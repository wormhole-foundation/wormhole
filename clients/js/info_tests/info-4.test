info contract mainnet Bsc NFTBridge
