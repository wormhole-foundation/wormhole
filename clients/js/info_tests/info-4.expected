0x5a58505a96D1dbf8dF91cB21B54419FC36e93fdE
